### DISCUSSION TITLE
Different language support and Multilingual page support

### DISCUSSION THREAD

**Discussion by chun2088** [GitbookIO/community#552] [Feature Requests]
It is very hard to write pages with language other than alphabet due to inputs backtracking and skipping lines and moving to different positions 
especially when using Asian languages. 

![image](https://github.com/GitbookIO/community/assets/75701828/252f7f3c-fa14-4b11-9347-5534b79f8c79)
Please have feature to have a copy of the pages in different languages like how wiki does it ( features like how link redirects to english page if same language is not supported)

**Comment by MadeByPP** [comment]
+1

I write documents using the Korean keyboard. As mentioned in the text, we are experiencing the same problem of input being traced back or skipping lines and moving to a different location.

**Comment by derloos** [comment]
@chun2088 How does writing it elsewhere, then pasting work for you?
